pipelines/plain_pipelines/paas-hackmd.yml

```diff
@@ -0,0 +1,66 @@
+---
+resource_types:
+  - name: s3-iam
+    type: docker-image
+    source:
+      repository: governmentpaas/s3-resource
+      tag: fda60bf4c5f85e96c16f704e128e5ead9e84d30d
```

Comment:
I don't see this pipeline using any S3 resources. Is this needed?

Reply:
That would be because I copy and pasted from the product page deployment pipeline. It can be replaced with something more sensible.

Reply:
> That would be because I copy and pasted from the product page deployment pipeline. It can be replaced with something more sensible.

By that I mean removed. D'oh.

Removed.